lib/api/src/lib.rs

```diff
@@ -1,4 +1,8 @@
 //! Wasmer API
+#![doc(
+    html_logo_url = "https://avatars3.githubusercontent.com/u/44205449?s=200&v=4",
```

Comment:
```suggestion
    html_logo_url = "https://github.com/wasmerio.png?size=200",
```

That's a pro-tip, but I think it's better to use that URL to save us stupid maintenance in the future. Thoughts?